src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -39,6 +40,13 @@ export class HostsComponent implements OnInit {
         pipe: this.cephShortVersionPipe
       }
     ];
+    this.interval = setInterval(() => {
```

Comment:
Does it make sense to add a mechanism which makes sure that the data is not fetched during another fetch is in progress?

Reply:
@votdev I agree, but we have the same problem on all other pages. Shouldn't we merge this PR as is, and I will fix the "stacked calls" problem on a separate PR for every page?

Reply:
Ignore my previous comment, I will fix the hosts page on this PR, and others will be fixed on a separate PR.

Reply:
OK, i'm fine with both proposals.

Reply:
We can use your improvements for this PR as a template for the other pages then.

Reply:
@votdev Now I'm preventing multiple executions of the fetch data in parallel. 

I've also added a commit that prevent double click on datatable refresh button.